Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/PipelineSwitchingTest.java

```diff
@@ -0,0 +1,41 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+import org.firstinspires.ftc.team6220_PowerPlay.GrabberCameraPipeline;
+import org.firstinspires.ftc.team6220_PowerPlay.RobotCameraPipeline;
+import org.openftc.easyopencv.OpenCvCamera;
+import org.openftc.easyopencv.OpenCvCameraFactory;
+import org.openftc.easyopencv.OpenCvCameraRotation;
+
+@Autonomous(name = "PipelineSwitchingTest")
+public class PipelineSwitchingTest extends BaseAutonomous
+{
+    @Override
+    public void runOpMode() throws InterruptedException
+    {
+        // creates camera streams
+        int cameraMonitorViewId = hardwareMap.appContext.getResources().getIdentifier("cameraMonitorViewId", "id", hardwareMap.appContext.getPackageName());
+        // creates cameras
+        robotCamera = OpenCvCameraFactory.getInstance().createWebcam(hardwareMap.get(WebcamName.class, "RobotCamera"), cameraMonitorViewId);
+        // creates pipelines
+        GrabberCameraPipeline grabberCameraPipeline = new GrabberCameraPipeline();
+        RobotCameraPipeline robotCameraPipeline = new RobotCameraPipeline();
+        grabberCameraPipeline.setRanges(Constants.LOWER_YELLOW, Constants.UPPER_YELLOW);
+        robotCameraPipeline.setRanges(Constants.LOWER_BLUE, Constants.UPPER_BLUE);
+        // starts streaming camera
+        startCameraWithPipeline(grabberCameraPipeline, robotCamera, Constants.CAMERA_X, Constants.CAMERA_Y);
+        waitForStart();
+        robotCamera.closeCameraDevice();
+        while (opModeIsActive()){
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
```suggestion
        while (opModeIsActive()) {
```